frontend/src/components/panels/quest-panel.tsx

```diff
@@ -393,6 +426,17 @@ export const QuestPanel: FunctionComponent<QuestPanelProps> = ({
 
     return (
         <StyledQuestPanel className="no-scrollbars">
+            {zone?.autoquests.map(
+                (autoquest) =>
+                    !acceptedQuests.find((quest) => quest.node.id === autoquest.id) && (
+                        <AutoQuestItem
+                            key={autoquest.id}
+                            id={autoquest.id || ''}
+                            questName={autoquest.name?.value || ''}
+                            player={player}
+                        />
+                    )
+            )}
```

Comment:
non-blocking: this is a little hard to read... I think this might be easier to read if it filtered the autoquests list, rather than conditionally returning either the AutoQuestItem or undefined .... something like:

```suggestion
            {zone?.autoquests.filter((quest) => acceptedQuests.some((quest) => quest.node.id === autoquest.id)).map((autoquest) => (
                        <AutoQuestItem
                            key={autoquest.id}
                            id={autoquest.id || ''}
                            questName={autoquest.name?.value || ''}
                            player={player}
                        />
                    )
            )}
```